src/Microsoft.ML.Transforms/Text/TextCatalog.cs

```diff
@@ -30,24 +29,23 @@ public static class TextCatalog
         /// </example>
         public static TextFeaturizingEstimator FeaturizeText(this TransformsCatalog.TextTransforms catalog,
             string outputColumnName,
-            string inputColumnName = null,
-            Action<TextFeaturizingEstimator.Settings> advancedSettings = null)
+            string inputColumnName = null)
             => new TextFeaturizingEstimator(Contracts.CheckRef(catalog, nameof(catalog)).GetEnvironment(),
-                outputColumnName, inputColumnName, advancedSettings);
+                outputColumnName, inputColumnName);
 
         /// <summary>
         /// Transform several text columns into featurized float array that represents counts of ngrams and char-grams.
         /// </summary>
         /// <param name="catalog">The text-related transform's catalog.</param>
         /// <param name="outputColumnName">Name of the column resulting from the transformation of <paramref name="inputColumnNames"/>.</param>
         /// <param name="inputColumnNames">Name of the columns to transform. If set to <see langword="null"/>, the value of the <paramref name="outputColumnName"/> will be used as source.</param>
-        /// <param name="advancedSettings">Advanced transform settings</param>
+        /// <param name="options">Advanced transform settings.</param>
         public static TextFeaturizingEstimator FeaturizeText(this TransformsCatalog.TextTransforms catalog,
             string outputColumnName,
             IEnumerable<string> inputColumnNames,
```

Comment:
>IEnumerable<string> inputColumnNames [](start = 12, length = 36)

I know it is not your change, but can we go with params, rather than an IEnumerable?

Looking at the sample, having to construct a List is overkill.

Reply:
Created issue #2460 to solicit suggestions on proper  API for this. 

Would not want to block this PR in the meantime.

---
In reply to: [254357200](https://github.com/dotnet/machinelearning/pull/2394#discussion_r254357200) [](ancestors = 254357200)